tests/xbot/common/command/SetpointSystemTest.java

```diff
@@ -0,0 +1,46 @@
+package xbot.common.command;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.wpi.first.wpilibj.command.Scheduler;
+import xbot.common.injection.BaseWPITest;
+
+public class SetpointSystemTest extends BaseWPITest{
+
+    @Before
+    public void setUp() {
+        super.setUp();
```

Comment:
Thought I needed it, or it wouldn't call it on the base class - but just tested it now and it works! So, removing it.